src/App.js

```diff
@@ -1,24 +1,234 @@
+        <FeaturedCard />
+        <TextArticle
+          title="Our services"
+          text="We provide to you the best choiches for you. Adjust it to your health needs and make sure your undergo treatment with our highly qualified doctors you can consult with us which type of service is suitable for your health"
+          centerText={true}
+        />
+
+        <div className="relative contents">
+          <LandingIcon type="wave" className="absolute left-0 -z-10 -mt-10 lg:w-[60vw] w-[90vw]" />
+          <LandingIcon
+            type="fullDotts"
+            className="absolute right-32 mt-80 -z-10"
+          />
+          <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 grid-rows-2 justify-items-center gap-y-10 gap-x-6">
+            <MiniCard
+              iconType="search"
+              title="Search doctor"
+              description="Choose your doctor from thousands of specialist, general, and trusted hospitals"
+            />
+            <MiniCard
+              iconType="pharmacy"
+              title="Online pharmacy"
+              description="Buy  your medicines with our mobile application with a simple delivery system"
+            />
+            <MiniCard
+              iconType="consultation"
+              title="Consultation"
+              description="Free consultation with our trusted doctors and get the best recomendations"
+            />
+            <MiniCard
+              iconType="details"
+              title="Details info"
+              description="Free consultation with our trusted doctors and get the best recomendations"
+            />
+            <MiniCard
+              iconType="emergencyCare"
+              title="Emergency care"
+              description="You can get 24/7 urgent care for yourself or your children and your lovely family"
+            />
+            <MiniCard
+              iconType="tracking"
+              title="Tracking"
+              description="Track and save your medical history and health data"
+            />
+          </div>
+        </div>
+        <div className="flex justify-center mt-[10vw] mb-[20vw] ">
+          <Button
+            bgColor="bg-white"
+            hoverColor="hover:bg-primary-blue hover:text-white"
+            textColor="text-primary-blue"
+            border="border-2 border-primary-blue "
+          >
+            Learn more
+          </Button>
+        </div>
+        <div className="grid lg:grid-cols-2 grid-cols-1">
+          <img
+            src={articleIllustration02}
+            className="w-full"
+            alt="second article"
+          />
+          <div className="lg:pl-40 lg:pr-20">
+            <TextArticle
+              title="Leading healthcare providers"
+              text="Trafalgar provides progressive, and affordable healthcare, accessible on mobile and online for everyone. To us, it’s not just work. We take pride in the solutions we deliver"
+              centerText={false}
```

Comment:
generally, boolean flags like this tend to accumulate overtime, and turn into chaos. allowing to pass classnames to the component generally gives us all the power and flexibility we may need